test/integration/test-cases.yaml

```
@@ -202,3 +251,17 @@ tests:
   update universal login branding prompts (mfa-push):
     command: cat ./test/integration/fixtures/update-ul-prompts-mfa-push.json | auth0 ul prompts update mfa-push
     exit-code: 0
+
+  tenants use:
+    command: auth0 tenants use $AUTH0_CLI_CLIENT_DOMAIN
+    exit-code: 0
+    stderr:
+      contains: 
+        - "Default tenant switched to:"
+
+  tenants open:
+    command: auth0 tenants open $AUTH0_CLI_CLIENT_DOMAIN --no-input
+    exit-code: 0
+    stderr:
+      contains: 
+        - "Open the following URL in a browser:"
```

I think it would be nice to have a soft assertion that an actual URL is printed out. Maybe something like:

```suggestion
        - "Open the following URL in a browser: https://manage.auth0.com/dashboard/"
```